mlflow/types/llm.py

```diff
@@ -159,12 +159,12 @@ class ToolCall(_BaseDataclass):
     Args:
         function (:py:class:`FunctionToolCallArguments`): The arguments of the function tool call.
         id (str): The ID of the tool call. Defaults to a random UUID.
-        type (str): The type of the object. Currently only "function" is supported.
+        type (str): The type of the object.
```

Comment:
```suggestion
        type (str): The type of the object. Defaults to "function"
```

nit: to better communicate the default value